extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+class DiscussionsDataService {
```

Comment:
You're probably correct, but I think given we are currently just using it to run AB Tests it might make sense to keep it within this extension for now. At least until it becomes a more permanent thing